src/workspace-mapping.ts

```diff
@@ -67,6 +67,7 @@ export function handleWorkspaceUpdated(event: WorkspaceUpdated): void {
   if(json.logoIpfsHash) entity.logoIpfsHash = json.logoIpfsHash!
   if(json.coverImageIpfsHash) entity.coverImageIpfsHash = json.coverImageIpfsHash
   if(json.socials) entity.socials = mapWorkspaceSocials(entityId, json.socials!)
+  if(json.tokens) entity.tokens = mapWorkspaceTokens(entity.id, json.tokens!)
```

Comment:
We went over this in the meet as well, an array populated by `derivedFrom` must not be set in the code

Reply:
I agree with you here, but in the last code pushed by you, `// TODO: map tokens and save` was mentioned here in line 70, that's why did this. Else I do believe, we shouldn't set the token here

Reply:
I think there was some misunderstanding -- I had said `map tokens and save`, not to set `entity.tokens`